docs/ubuntu-installation-guide.md

```diff
@@ -25,7 +25,7 @@ For more information on installing Docker please refer to the
 
 ```
 $ sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/xUbuntu_$(lsb_release -rs)/ /' >> /etc/apt/sources.list.d/clear-containers.list"
-$ curl -fsSL http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/xUbuntu_$(lsb_release -rs)/Release.key | sudo apt-key add -
+$ wget -qO - http://download.opensuse.org/repositories/home:/clearcontainers:/clear-containers-3/xUbuntu_$(lsb_release -rs)/Release.key | sudo apt-key add -
 $ sudo -E apt-get update
 $ sudo -E apt-get -y install -y cc-runtime cc-proxy cc-shim
```

Comment:
Sorry to be a pain, but could you remove the 2nd `-y` in this line in either the same commit or a new one (this was my fault I think! :)

Reply:
@jodh-intel done :)

Reply:
Thanks very much! ;)